scripts/first_run.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
```

Comment:
let's use `/bin/bash`. Otherwise wwe will at some point have to change statements like
```bash
export a=b
```
to
```bash
a=b
export a
```